### PR TITLE
Fix build issues

### PR DESCRIPTION
**Pull request by xantronix**
Fix build issues due to missing dependency declarations, as well as an
invalid, off-by-one test plan

Other changes:

    * Bump version to 0.94

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Revision history for Unix-PID-Tiny
 
+0.94 Wed Apr 24 16:56:09 2019
+    - declare Test::Warn, File::Slurp, File::Temp dependencies
```
Comment (drmuey) [inline Changes:4]: I don’t see anything about Test::Warn, File::Slurp, File::Temp in e37c8ab6296e558acbf336a2d4e7b71649ad0026 did something not get pushed?

File: lib/Unix/PID/Tiny.pm [hunk 2]
```diff
@@ -3,7 +3,7 @@ package Unix::PID::Tiny;
 use strict;
 use warnings;
 
-our $VERSION = '0.93';
+our $VERSION = '0.94';
```
Comment (drmuey) [inline lib/Unix/PID/Tiny.pm:6]: Can you bump this in the README and .pod file too? thanks!

**Review by drmuey — CHANGES_REQUESTED** [review]

File: Changes (hunk 1 above)
Comment (xantronix) [reply Changes:4]: We were getting RPM build errors related to missing Test::Warn, File::Slurp and File::Temp.

File: lib/Unix/PID/Tiny.pm (hunk 2 above)
Comment (xantronix) [reply lib/Unix/PID/Tiny.pm:6]: My bad! Sure. :)

File: Changes (hunk 1 above)
Comment (drmuey) [reply Changes:4]: yeah, that I gathered, I just didn't see any change relating to those modules in e37c8ab whcih made me think something is missing. I see them now in ce81f0c 👍

File: Makefile.PL [hunk 3]
```diff
@@ -8,10 +8,16 @@ WriteMakefile(
     ABSTRACT_FROM => 'lib/Unix/PID/Tiny.pm',
     PL_FILES      => {},
     PREREQ_PM     => {
-        'Test::More'     => 0,
-        'Test::MockFile' => 0,
-        'File::Slurp'    => 0
+        'File::Slurp' => 0,
```
Comment (drmuey) [inline Makefile.PL:11]: this is also for tests only

File: Makefile.PL [hunk 4]
```diff
@@ -8,10 +8,16 @@ WriteMakefile(
...
+        'File::Temp'  => 0
```
Comment (drmuey) [inline Makefile.PL:12]: this is also for tests only

File: Makefile.PL (hunk 3 above)
Comment (drmuey) [reply Makefile.PL:11]: I'll do it afterward 👍

File: Makefile.PL (hunk 4 above)
Comment (drmuey) [reply Makefile.PL:12]: I'll do it afterward 👍

**Review by drmuey — APPROVED** [review]

**Pull request merged**